### ISSUE TITLE
No ID provided. An ID must be given when creating a grid.

### ISSUE THREAD

**Issue by Marchrius** [patbuergin/angular-widget-grid#14]
This error occur when using ui-grid and angular-widget-grid in same project.

The `Grid` factory will be replaced by `ui.grid.Grid`

**Comment by W-prog** [comment]
Hello, 
I have the same error even if I load the ui-grid after the angular-widget-grid
Have you an issue ?

**Comment by Marchrius** [comment]
@W-prog no, I solved the issue by loading widgetGrid before ui.grid.

It will be better if the maintainer changes the object name with a _prefix_ or something else.

**Comment by W-prog** [comment]
Oh yeah, i've trying load widget-grid before but the error is more large
for example : 
TypeError: Cannot read property 'length' of null
    at ui-grid.js:3277

Yes for "It will be better if the maintainer changes the object name with a prefix or something else"
Have you an idea where in librairie I do that  ?
I see on line 161

vm.grid = new Grid($scope.rows, $scope.columns);
For rename the var Grid but i think i do rename all on the lib

**Comment by Marchrius** [comment]
Yes, you have to replace all the occurrences of "Grid" into the lib.

**Comment by sebius8780** [comment]
Hello guys !
Thanks for the tip, we actually needed to change Grid to someNameGrid AND gridUtil to someNameGridUtil.

Just saying in case someone needed it !

**Comment by Marchrius** [comment]
@W-prog @sebius8780 At the moment I've found a work around for this (I forgotten it). 

#13 In this PR there is a commit about force inject the correct Grid.
